hw/top_earlgrey/rdc/chip_earlgrey_asic_scenario.tcl

```diff
@@ -13,9 +13,12 @@
 
 # POR_N
 # When POR_N released, POK remains high (already released from reset)
+# dio_pad_attr_q[12] is SCK PAD. The invert signal remains 0 always
```

Comment:
The pad attributes could technically be set because they are connected to registers - but I presume this is not a valid use case, so LGTM!